### ISSUE TITLE
Damage Meter

### ISSUE THREAD

**Issue by Battler624** [goatcorp/suggestions#492]
### Description
A Meter with multiple modules meant to display various types of details.

### More info
Something like recount or details from wow or in FFXIV, something like ACT but I wish for it to be contained within dalamundplugins without something external (So seperate from ACT if possible)

----

 - [X] I checked if this request wasn't already proposed (the terms i used while searching are : dps, damage, tracker, act, meter)
 - [X] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher#isnt-this-cheating)

**Comment by Eternita-S** [comment]
Very unlikely to happen as it's literally reinventing the wheel. Parsing plugin is quite complex and it won't be easy to remake it into dalamud. 
Also consider the fact that parsing plugin is barely useful on it's own. With it you'd need to port at least overlayplugin, cactbot and triggernometry to get it more or less useful. And the amount of work needs to be done in order to do so... is simply scary. 
Consider also the fact that there are many other ACT plugins that are supposed to work together with ffxiv parsing plugin. Consider the fact that you'd need to get fflogs approvement for your plugin as well to be able to upload logs. 
Basically there is way, way too much work needs to be done to even think of starting to work on parsing plugin for dalamud, to put it simple.

**Comment by Battler624** [comment]
> Very unlikely to happen as it's literally reinventing the wheel. Parsing plugin is quite complex and it won't be easy to remake it into dalamud.
> Also consider the fact that parsing plugin is barely useful on it's own. With it you'd need to port at least overlayplugin, cactbot and triggernometry to make it more or less useful. And the amount of work needs to be done in order to do so... is simply scary.
> Consider also the fact that there are many other ACT plugins that are supposed to work together with ffxiv parsing plugin. Consider the fact that you'd need to get fflogs approvement for your plugin as well to be able to upload logs.
> Basically there is way, way too much work needs to be done to even think of starting to work on parsing plugin for dalamud, to put it simple.

The reason I wish for this is simple, it would be easier for this to work on linux for those of us that use linux.

I am honestly transitioning myself and quicklauncher is so freakin great which I am really grateful for, ACT requires a lot of work and breaks every once in a while, I am hoping that if its implemented within dalamund it would be so perfect.

For my use case I honestly dont need uploadability to FFLogs.
